website/screens/components/status-light/usage/StatusLightUsagePage.tsx

```diff
@@ -0,0 +1,118 @@
+import {
+  DxcParagraph,
+  DxcBulletedList,
+  DxcFlex,
+} from "@dxc-technology/halstack-react";
+import QuickNavContainer from "@/common/QuickNavContainer";
+import QuickNavContainerLayout from "@/common/QuickNavContainerLayout";
+import DocFooter from "@/common/DocFooter";
+import Figure from "@/common/Figure";
+import Image from "@/common/Image";
+import variants from "./images/status-light_variants.jpg";
+
+const sections = [
+  {
+    title: "Usage",
+    content: (
+      <DxcParagraph>
+        The Status light component serves as a powerful communicator of an
+        element or process' status. It focuses exclusively on conveying semantic
+        information within a range of five variants, each of them aligned with
+        its specific process status.
+      </DxcParagraph>
+    ),
+    subSections: [
+      {
+        title: "Do's",
+        content: (
+          <DxcBulletedList>
+            <DxcBulletedList.Item>
+              Match each status light variant with the correct semantic meaning
+              to communicate clearly and consistently messages about our
+              statuses.
+            </DxcBulletedList.Item>
+            <DxcBulletedList.Item>
+              Use each size of the component according to the different screen
+              sizes where it will be placed, paying special attention to
+              maintaining legibility and functionality.
+            </DxcBulletedList.Item>
+            <DxcBulletedList.Item>
+              Use concise and precise labels that reflect the status of a
+              process accordingly. You should not use more than three words in a
+              status light label.
+            </DxcBulletedList.Item>
+          </DxcBulletedList>
+        ),
+      },
+      {
+        title: "Don'ts",
+        content: (
+          <DxcBulletedList>
+            <DxcBulletedList.Item>
+              Mix semantic meanings within Status lights, as this will lead to
+              confusion and misinterpretation of process states.
+            </DxcBulletedList.Item>
+            <DxcBulletedList.Item>
+              Use Status light and semantic badges in the same data
+              visualization format (tables, charts, data grids...).
+            </DxcBulletedList.Item>
+            <DxcBulletedList.Item>
+              Overuse the component. In screens with high cognitive load, you
+              should not overuse this component, as it may confuse users and
+              interrupt the reading flow.
+            </DxcBulletedList.Item>
+          </DxcBulletedList>
+        ),
+      },
+    ],
+  },
+  {
+    title: "Variants",
+    content: (
+      <>
+        <DxcParagraph>
+          The Status light component has five different modes, each one
```

Comment:
For consistency purposes, I think we should keep the same approach as with the [Button variants](https://developer.dxc.com/halstack/next/components/button/usage/#variants) or the [Alert variants](https://developer.dxc.com/halstack/next/components/alert/usage/).

1. Use a table with a row for each variant.
2. Instead of using an image, I think it's better to use coded examples as in the rest of the Usage pages.